deploy/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
```

Comment:
let's also have bash report what command is running.  `set -ex`

Reply:
done